### ISSUE TITLE
Send email when referral has been paid

### ISSUE THREAD

**Issue by sumobi** [AffiliateWP/AffiliateWP#1687]
We should really have an email that is sent to affiliates when they have been paid. Currently this is only possible via a code snippet:

https://github.com/AffiliateWP/affiliatewp-code-snippet-library/blob/master/emails/email-affiliate-when-referral-paid.php

**Comment by pippinsplugins** [comment]
I don't think it makes sense to do this on a per-referral basis as that could result in numerous notifications in a single payout. Instead, let's do it when a payout log is created.

**Comment by sumobi** [comment]
Absolutely, great point.
